mlflow/environment_variables.py

```diff
@@ -156,3 +156,11 @@ def get(self):
 #: for more information.
 #: (default: ``False``)
 MLFLOW_SQLALCHEMYSTORE_ECHO = _BooleanEnvironmentVariable("MLFLOW_SQLALCHEMYSTORE_ECHO", False)
+
+#: Specifies the ``poolclass`` parameter to use for ``sqlalchemy.create_engine`` in the
+#: SQLAlchemy tracking store. See https://docs.sqlalchemy.org/en/14/core/engines.html#sqlalchemy.create_engine.params.poolclass
+#: for more information.
+#: (default: ``None``)
+MLFLOW_SQLALCHEMYSTORE_POOLCLASS = _EnvironmentVariable(
+    "MLFLOW_SQLALCHEMYSTORE_POOLCLASS", str, None
```

Comment:
What's the default pool do we use right now? Is it QueuePool? Maybe just use that as default value for the env var here so that users won't be confused about what `None` means.

Reply:
None means the parameter is not set. And it's the default when create_pool is called.

Reply:
We are following the default of sqlalchemy and other environment parameters has the same convention. There's no need to always set it.